test/SIL/Parser/basic2_noncopyable_generics.sil

```diff
@@ -26,8 +26,12 @@ struct NCG<T> : ~Copyable {
   deinit
 }
 
-// CHECK-LABEL: struct NEG<T> : ~Escapable {
+// TODO: Fix SIL printing of the type variable below:
+// CHECK-LABEL: struct NEG<T> : ~Escapable where T : ~Escapable {
```

Comment:
What's wrong with the printing?

Reply:
I was expecting `struct NEG<T : ~Escapable> : ~Escapable {` instead of `struct NEG<T> : ~Escapable where T : ~Escapable {`

Reply:
Both forms are equivalent and the ASTPrinter is not wrong to always prefer the `where` clause form.

Reply:
okay!